src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -983,11 +953,11 @@
   ;;   Ensuring producer_timestamp is NOT NULL
   ;;   Changing the types of hashes and uuids in postgres to bytea and uuid respectively
   []
-  (let [hash-type (if (sutils/postgres?) "bytea" "varchar(40)")
-        uuid-type (if (sutils/postgres?) "uuid" "varchar(255)")
-        json-type (if (sutils/postgres?) "json" "text")
-        munge-hash (if (sutils/postgres?) (fn [column] (format "('\\x' || %s)::bytea" column)) identity)
-        munge-uuid (if (sutils/postgres?) (fn [column] (format "%s::uuid" column)) identity)]
+  (let [hash-type "bytea"
+        uuid-type "uuid"
+        json-type "json"
```

Comment:
Oh, right - I converted this code first, then just started replacing these values in the SQL, but didn't get back to these.